index.html

```diff
@@ -0,0 +1,571 @@
+
+## The Basics
+
+Core schemas:
+  1. **must** be valid GraphQL schema documents,
+  2. **must** contain exactly one `SchemaDefinition`, and
+  3. **must** use the `@core` directive on their schema definition to declare any features they reference by using [`@core(feature:)`](#core) to reference a [well-formed feature URL](#feature-urls).
+
+The first `@core` directive on the schema **must** reference the core spec itself, i.e. this document.
+
+<view-of src=basic.graphql title='Basic core schema using @core and @example'></view-of>
+
+### Unspecified directives are passed through
+
+Existing schemas likely contain definitions for directives which are not versioned, have no specification document, and are intended mainly to be passed through. This is the default behavior for core schema processors:
+
+```graphql
+#::[Unspecified directives are passed through]
+schema
+  @core(feature: "https://lib.apollo.dev/core/v0.1")
+{
+  query: Query
+}
+
+type SomeType {
+  field: Int @another
+}
+
+# `@another` is unspecified. Core processors will not extract metadata from
+# it, but its definition and all usages within the schema will be exposed
+# in the export schema.
+directive @another on FIELD
+```
+
+### Renaming `@core` itself
+
+It is possible to rename the `@core` directive itself with the same `as:` mechanism used for all features:
+
+```graphql
+#::[Renaming `@core` to `@coreSchema`]
+schema
+  @coreSchema(feature: "https://lib.apollo.dev/core/v0.1", as: "coreSchema")
+  @coreSchema(feature: "https://example.com/example/v1.0")
+{
+  query: Query
+}
+
+type SomeType {
+  field: Int @example
+}
+
+directive @example on FIELD
+```
+
+## `@core`
+
+```graphql
+#::[The `@core` directive]
+directive @core(feature: core__FeatureUrl!, as: String, export: Boolean)
+  repeatable on SCHEMA
+```
+
+Declare a core feature present in this schema.
+
+Documents **must** include a definition for the `@core` directive. The provided definition must be *compatible* with the definition above, but may:
+- **Omit optional arguments** if they are never used in the document,
+- **Omit locations** where the directive never occurs,
+- **Introduce new directive locations**, the behavior of which is unspecified here.
+- **Introduce new arguments.** New arguments **must** be prefixed, e.g. `example__extensionArgument: Bool`. The prefix **must** be the name of a feature referenced with this or another `@core` directive within the document.
+- **Use `String` in place of custom scalars.** This is an ease-of-authoring affordance which allows document authors to omit the definitions of custom scalar types provided by features. Processors **must** continue to interpret such arguments and fields according to the encoding and decoding rules of the custom scalar type. For example, the `@core` directive may be defined as follows, with no change to the behavior:
+
+```graphql
+#::[The `@core` directive, defined with `feature: String`]
+directive @core(feature: String)
```

Comment:
This is missing the `!` so this now lets you write `@core()` on a SCHEMA which seems not in fact compatible?